packages/components/src/schemas/HomePageAPI.schema.ts

```diff
@@ -0,0 +1,12 @@
+import { Quote } from './Quote.schema';
+import { Article } from './Article.schema';
+import { Diary } from './Diary.schema';
+import { User } from './User.schema';
+
+export interface HomePageInitSchema {
+  quotes: Quote[];
+  articles: Article[];
+  diaries: Diary[];
+  challenges: any;
```

Comment:
dont use '`any`', create a dummy schema for time being, but don't use `any`

Reply:
abhi decided nhi hai, kuch kia toh backned se init call fatjayegi
added a todo